apis/pagesapi/pagesapi.yaml

```diff
@@ -347,62 +267,6 @@ paths:
             $ref: '#/definitions/ContentCreateErrorResponse'
         '500':
           description: 'INTERNAL SERVER ERROR ! Looks like something went wrong! These errors are tracked automatically, but if the problem persists feel free to contact us. In the meantime, try refreshing.'

```

Comment:
INTERNAL SERVER ERROR, we are tracking the error and cater to it on priority. Try to refresh the page and if the problem persists contact us at info@sunbird.org